README.md

```diff
@@ -14,6 +14,9 @@ SharpToken is a C# library that serves as a port of the Python [tiktoken](https:
 It provides functionality for encoding and decoding tokens using GPT-based encodings. This library is built for .NET 6, .NET 8
 and .NET Standard 2.0, making it compatible with a wide range of frameworks.
 
+> [!Important]
+> The functionality in `SharpToken` has been added to [`Microsoft.ML.Tokenizers`](https://www.nuget.org/packages/Microsoft.ML.Tokenizers). `Microsoft.ML.Tokenizers` is a tokenizer library being developed by the .NET team and going forward, the central place for tokenizer development in .NET. By using `Microsoft.ML.Tokenizers`, you should see improved performance over existing tokenizer library implementations, including `SharpToken`. A stable release of `Microsoft.ML.Tokenizers` is expected alongside the .NET 9.0 release (November 2024). Instructions for migration can be found at https://github.com/dotnet/machinelearning/blob/main/docs/code/microsoft-ml-tokenizers-migration-guide.md.
```

Comment:
should we specify the `Tiktoken` tokenizer to ensure users will not get lost which tokenizer to use when replacing SharpToken? I see the benchmark code has it but maybe not that obvious to the users? ` Tokenizer.CreateTiktokenForModel("gpt-4");`

Reply:
Sure, I just copied this from the text that @luisquintanilla helped draft for DeepDev.  We can mention `Tiktoken`